bitnami/kafka/README.md

```diff
@@ -178,7 +178,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `podSecurityContext.enabled`         | Enable security context for the pods                                                                                                                                                          | `true`          |
 | `podSecurityContext.fsGroup`         | Group ID for the filesystem used by the containers                                                                                                                                            | `1001`          |
 | `podSecurityContext.runAsUser`       | User ID for the service user running the pod                                                                                                                                                  | `1001`          |
-| `containerSecurityContext`           | Kafka containers' Security Context                                                                                                                                                            | `{}`            |
+| `containerSecurityContext.enabled`   | Kafka containers' Security Context                                                                                                                                                            | `false`         |
```

Comment:
This description doesn't match the one you indicated in the values metadata (using `## @param  ... `)
Please use the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool to ensure the README.md tables are synced with the values metadata.

Reply:
Should be fine now: no errors reported by `readme-generator`.